rs/sns/governance/src/governance.rs

```diff
@@ -4749,11 +4781,13 @@ impl Governance {
                 upgrade_steps: cached_upgrade_steps.upgrade_steps,
                 response_timestamp_seconds: cached_upgrade_steps.response_timestamp_seconds,
                 target_version: self.proto.target_version.clone(),
+                journal: self.proto.upgrade_journal.clone(),
```

Comment:
Can we limit the size of this response? Feel free to add a `TODO[Jira-Ticket] ...`

Reply:
Before we ship this, all clones and responses should be bounded.

Reply:
Done!